islandora_pathauto.module

```diff
@@ -319,3 +321,88 @@ class IslandoraFedoraObjectReadOnly implements ArrayAccess {
    */
   public function offsetUnset($offset) {}
 }
+
+/*
+ * Implements hook_form_alter()
+ * 
+ * Adds a TextField to Islandora Properties so the User can Change
+ * the URL Alias.
+ */
+function islandora_pathauto_form_alter(&$form , &$form_state, $form_id){
+  if ( $form_id == 'islandora_object_properties_form' ){
+    
+    if ( !pathauto_pattern_load_by_entity('islandora') ){
```

Comment:
Will this still work if there is no default pattern for "islandora", but there are patterns for specific content models under "islandora"?